airbyte-webapp/src/hooks/services/useConnectorAuth.tsx

```diff
@@ -105,18 +111,46 @@ export function useRunOauthFlow(
   done?: boolean;
   run: () => void;
 } {
-  const { values, setValues } = useFormikContext();
+  const {
+    values,
+    setValues,
+    validateForm,
+    setFieldTouched,
+  } = useFormikContext();
+  const oAuthFieldsToValidate = useMemo(() => {
+    const oAuthRequired =
+      connector?.authSpecification?.oauth2Specification?.oauthFlowInputFields;
+    return oAuthRequired ? oAuthRequired.map((value) => value.join(".")) : [];
```

Comment:
:art: Suggestions for a bit better code style:

```suggestion
    return oAuthRequired?.map((value) => value.join(".")) ?? [];
```

Reply:
Thanks @timroes, but I have already applied same changes in separate branch that will be merged into this one. No need to make same changes twice :)